core/src/test/java/org/elasticsearch/index/store/CorruptedFileIT.java

```diff
@@ -568,34 +590,21 @@ private int numShards(String... index) {
         return shardIterators.size();
     }
 
-    private Map<String, List<Path>> findFilesToCorruptForReplica() throws IOException {
-        Map<String, List<Path>> filesToNodes = new HashMap<>();
-        ClusterState state = client().admin().cluster().prepareState().get().getState();
-        Index test = state.metaData().index("test").getIndex();
-        for (ShardRouting shardRouting : state.getRoutingTable().allShards("test")) {
-            if (shardRouting.primary()) {
```

Comment:
after talking this through offline I am convinced we didn't loose anthing here. +1 to proceed